server/build/babel/plugins/handle-static.js

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs')
+const path = require('path')
+const md5 = require('md5-file')
+
+const dev = process.env.NODE_ENV !== 'production'
+const moduleName = 'next/static'
+const staticStats = {}
+const staticStatsPath = process.env.__NEXT_STATIC_STATS_PATH__
+const staticDir = process.env.__NEXT_STATIC_DIR__
```

Comment:
need better way to tell babel plugin where the static dir is located

Reply:
@cloudkite I wonder if it would make sense for the static location to somehow be configured / set as part of config: https://github.com/zeit/next.js/blob/master/server/config.js

I know it's not configurable at the moment, but `getConfig` has access to `dir` and the result is cached / stored.

Reply:
@kochis that could work 👍  @arunoda what do you think of this approach?

Reply:
@kochis not sure when I'll get a chance to pick this up, feel free to jump in :)